skyrim-platform/src/platform_se/skyrim_platform/FridaHooks.cpp

```diff
@@ -25,6 +25,9 @@
 #include <sstream>
 #include <windows.h>
 
+#include <hooks/DInputHook.hpp>
```

Comment:
we include our (non-library) files with "" syntax (not <>)
and also please remove whitespace at line 27